content/03.results.md

```diff
@@ -100,7 +100,6 @@ We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and f
 Following SNV consensus calling (**Figure S1** and **Figure S2, panels A-F**), we observed lower expected tumor mutation burden (TMB) **Figure S2, panel G** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors only found within HGATs.  
```

Comment:
Not strictly related to PR but why not. Should we add in the Campbell 2017 reference here for defining hypermutant and ultrahypermutant?  The DOI is `10.1016/j.cell.2017.09.048`

Reply:
yes, good suggestion